impl/src/main/java/org/pentaho/platform/genericfile/providers/repository/RepositoryFileProvider.java

```diff
@@ -70,7 +73,7 @@ public class RepositoryFileProvider extends BaseGenericFileProvider<RepositoryFi
     }
   }
 
-  public static final String TYPE = "repository";
+  public static final String TYPE = "pur";
```

Comment:
I don't think this should change here. My original idea was to use `pur` on the PAS browse files bff side.